server/schedule/filters.go

```diff
@@ -31,6 +32,7 @@ type Filter interface {
 func FilterSource(opt Options, store *core.StoreInfo, filters []Filter) bool {
 	for _, filter := range filters {
 		if filter.FilterSource(opt, store) {
+			log.Debugf("[filter %T] filters store %v from source", filter, store)
```

Comment:
%T will output all the fields of the filter, is it ok here?

Reply:
no, it outputs like "*schedule.excludedFilter"